src/bvals/cc/bvals_cc_in_one.hpp

```diff
@@ -95,10 +96,12 @@ struct BndInfo {
   RefinementOp_t refinement_op = RefinementOp_t::None;
   Coordinates_t coords, coarse_coords; // coords
 
-  buf_pool_t<Real>::weak_t buf; // comm buffer from pool
-  ParArray6D<Real> var;         // data variable used for comms
-  ParArray6D<Real> fine;        // fine data variable for prolongation/restriction
-  ParArray6D<Real> coarse;      // coarse data variable for prolongation/restriction
+  buf_pool_t<Real>::weak_t buf;                   // comm buffer from pool
+  parthenon::ParArray6D<Real, VariableState> var; // data variable used for comms
+  parthenon::ParArray6D<Real, VariableState>
```

Comment:
I think that there shouldn't be type-casting going on because the `CellVariable`s now contain `ParArray6D<Real, VariableState>` as well. Nevertheless, type-casting from `ParArray*<Real>` to `ParArray*<Real, VariableState>` should work.